include/questdb/ingress/line_sender.h

```diff
@@ -171,6 +171,14 @@ line_sender_utf8 line_sender_utf8_assert(size_t len, const char* buf);
 #define QDB_UTF8_LITERAL(literal)                                              \
     line_sender_utf8_assert(sizeof(literal) - 1, (literal))
 
+/**
+ * Non-owning view of sender buffer.
```

Comment:
Add an extra note about safety here:
"Modifying the buffer will invalidate the borrowed buffer"